framework/helpers/BaseArrayHelper.php

```diff
@@ -247,6 +247,32 @@ public static function remove(&$array, $key, $default = null)
     }
 
     /**
+     * Search and removes an item from an array and returns the key if found. If the value does not exists it will return the default value.
+     *
+     * Example,
+     *
+     * ```php
+     * // $array = ['value1', 'value2'];
+     * $key = \yii\helpers\ArrayHelper::removeValue($array, 'value2');
+     * // result:
+     * // $array = ['value1']
```

Comment:
correct. I guess these // dont really add to the readability ;), but it is consistent with the other methods.